package/src/constructs/check.ts

```diff
@@ -59,7 +59,7 @@ export abstract class Check extends Construct {
     this.groupId = props.groupId
     // alertSettings, useGlobalAlertSettings, groupId, groupOrder
 
-    this.__checkFilePath = Session.checkFilePath
+    this.__checkFilePath = Session.checkFilePath!
```

Comment:
I think that sometimes the `Session.checkFilePath` will be undefined. For checks that come from `*.check.js` files, we should always have `Session.checkFilePath` but for browser checks that are created automatically from `*.spec.js` files, it isn't set.

https://github.com/checkly/checkly-cli/blob/3b9a21bffda3999c8bdaec1c2c04c17b8a0d2ad9/package/src/services/project-parser.ts#L60-L108

Reply:
Hmm. We gotta pass that each time then. I will take a look at it

Reply:
Sounds good 👍